cmd/create.go

```diff
@@ -23,8 +23,14 @@ var CreateCommand = cli.Command{
 			Value: "latest",
 			Usage: "District name",
 		},
+		cli.BoolTFlag{
```

Comment:
I added flag here because `This should probably apply to all ShowHeritage except for bcn app show?`
let me know what you think

Reply:
Hmm, it doesn't make sense for show because the user is deliberately requesting the information, so I don't think quiet is required here.

Reply:
ok!
I just removed the flags!
Please take a look when you have time